OmsAgent/omsagent.py

```diff
@@ -535,7 +535,9 @@ def enable():
 
     if HUtilObject is not None:
         if HUtilObject.is_seq_smaller():
-            return 0, "Current sequence number " + HUtilObject._context._seq_no + " is not greater than the sequence number of the most recent executed configuration, skipping enable."
+            log_output = "Current sequence number {0} is not greater than the sequence number of the most recent executed configuration, skipping enable.".format(HUtilObject._context._seq_no)
+            hutil_log_info(log_output)
+            return 0, log_output
```

Comment:
can you add this to AMA as well

Reply:
Made another PR for AMA: https://github.com/Azure/azure-linux-extensions/pull/1459